sample-templates/templateArcGISRuntimeSDKQt_Cpp/wizard.json

```
@@ -4,8 +4,8 @@
     "supportedProjectTypes": [ "Qt4ProjectManager.Qt4Project" ],
     "id": "A.Sample_ArcGISQt_CPP_100_14",
```

```suggestion
    "id": "A.Sample_ArcGISQt_CPP_200_1_0",
```

Similar change for other similar files.

---

well spotted, this has been hidden for some time already!